portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase

```diff
@@ -201,6 +205,7 @@ definition {
 	@priority = "3"
 	test SegmentOverviewDistributionFilteredByNumber {
 
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
```

Comment:
Hey @manuferreira, I think this validation can be done using other criteria.

This ticket is probable to have problems with automation because of https://issues.liferay.com/browse/LRAC-9329

I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket

Reply:
Thank you for highlighting LRAC-9329 @ViniLopes20. We will need to remember this issue until we can fully migrate to BQ.

I think it's okay to just disable these tests quickly to prevent failures, but as @ViniLopes20 said, a ticket should be created to refactor it.

Probably the easiest way to fix this would be to sync contacts with additional attributes if the required attribute type is not already in the default list of synced attributes.  I believe we can also create custom attributes to sync if we need any attribute types that are not in DXP by default.  Please message me if you have questions about this!

Reply:
I started to fix all these tests that need an update in this PR by creating a custom attribute on DXP (when it's necessary) this way we can maintain the idea of the test. 
Thanks, guys @christianstokes @ViniLopes20